pkg/registries/adapters/adapter.go

```diff
@@ -102,6 +102,8 @@ func imageToSpec(log *logging.Logger, req *http.Request) (*apb.Spec, error) {
 		return nil, err
 	}
 
+	spec.Image = spec.Image + ":" + apbtag
```

Comment:
Also something that popped in my head, what if `apbtag` is empty? this will create an image like:
```
mediawiki-apb:
```
I don't think that's valid.